test/beacon/Voucher.test.ts

```diff
@@ -644,6 +644,28 @@ describe('Voucher', function () {
                     ),
                 ).to.be.revertedWith('IexecPocoMock: Failed to sponsorMatchOrdersBoost');
             });
+
+            it('Should not match orders boost when SRLC transfer fails', async () => {
```

Comment:
Should we do it for _Classic_ too? (not sure of the answer)

Reply:
Was just for the coverage but we can

Reply:
Maybe use
```
            it('Classic', async () => await runTest(Classic));
            it('Boost', async () => await runTest(Boost));
```
?

Reply:
Done here : [added test for both classic and boost + todo](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/26/commits/e2fd847ab757baea170cfb286fa4fc62533248ec) and added a todo to do it on all match orders test for a next PR